fedora/main.fmf

```diff
@@ -0,0 +1,26 @@
+# CI configuration to run upstream tests in Fedora ecosystem
+# More info:
+# Testing framework: http://tmt.readthedocs.io/
+# CI system docs: https://packit.dev/testing-farm/
+
+summary: Run tests on Fedora Linux
+
+discover:
+    how: shell
+    tests:
+      - name: Upstream test suite
+        test: make test
+
+provision:
+    how: local
+
```

Comment:
```suggestion
```
I'd suggest removing this, as this is ignored by CI, usually left blank for regular use (to not make changes on users computer). _There should be in time, or is, a per-user config to change the default this for all instance._

By default it is `virtual` (using testcloud). In all cases, it can be overridden from command line using f.e.:
```
$ tmt run -a provision -h container
```
_Alternatively, `virtual`, `local`, or `minute` (internal)._

Reply:
This depends on your use-case, of course.

Reply:
@psss actually suggested this to me so that I can easily do `tmt run` instead of that `tmt run -a prov...`

Reply:
Yes, that's one use-case if the system is suitable (there are all required test dependencies), and packages to test installed. I'm just pointing out that for some scenarios some (destructive tests) encapsulation is preferred.